05-classes-oop/hw_05_classes_oop.py

```diff
@@ -0,0 +1,83 @@
+import sys
+from datetime import datetime
+
+
+class News:
+    def __init__(self, text, city):
+        self.type = "News"
+        self.text = text
+        self.city = city
+        self.date = datetime.now().strftime("%c")
+
+    def to_string(self):
+        return f"News: {self.text} | City: {self.city} | Date: {self.date} \n"
+
+
+class PrivateAd:
```

Comment:
The principle of inheritance is not used

Should be smth like:

class Publication:
    """Parent class for any manual publication"""
    def __init__(self):
        self.publication_text = input("Please enter the main text of your publication:\n")


class News(Publication):
    """Class for News manual input"""
    def __init__(self):
        super().__init__()
        self.city = input("Please enter the city:\n")
        self.publ_date = datetime.now()

    def build_news_publication(self):
        news = [{'type': 'News', 'text': self.publication_text, 'city': self.city, 'date': str(self.publ_date)}]
        return news